artemis/modules/nuclei.py

```diff
@@ -203,7 +236,11 @@ def _scan(
         if not targets:
             return []
 
-        templates_or_workflows_filtered = []
+        # Apply max_templates limit from configuration if set
+        if self.configuration and self.configuration.max_templates:
+            templates = templates[: self.configuration.max_templates]
```

Comment:
I don't think this is a good idea as it will skip the tempaltes quite randomly - I would vote for keeping severity as the only means of controlling how aggressive the module is